packages/components/package.json

```diff
@@ -30,7 +30,7 @@
     "build:styles": "postcss \"src/**/*.css\" --dir lib/ --base src/ --verbose",
     "build:watch": "yarn run build:js --watch",
     "build:storybook": "build-storybook -o storybook-static",
-    "chromatic": "chromatic --project-token=CHROMATIC_PROJECT_TOKEN --skip 'dependabot/**'",
+    "chromatic": "chromatic --only-changed",
```

Comment:
Interestingly, on the CI run chromatic actually ignored `--only-changed` because

> Could not retrieve changed files since baseline commit(s).
This typically happens after rebasing.

Something to keep an eye on after merging, to make sure it's actually working as expected.

Reply:
Might be because Turbosnap wasn't on for the main branch, but yes it is something to keep an eye out for

Reply:
Latest commit didn't have that message but rather
> Failed to retrieve dependent story files
🤔